python/foglamp/services/core/support.py

```diff
@@ -0,0 +1,253 @@
+# -*- coding: utf-8 -*-
```

Comment:
add hadhbang and move to scripts:

This way it can be used as py script (main) and REST Api can call SupportBuilder Object